src/models/event.ts

```diff
@@ -890,28 +882,6 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
         return this.decryptionPromise;
     }
 
-    /**
-     * Cancel any room key request for this event and resend another.
-     *
-     * @param crypto - crypto module
-     * @param userId - the user who received this event
-     *
-     * @returns a promise that resolves when the request is queued
-     */
-    public cancelAndResendKeyRequest(crypto: Crypto, userId: string): Promise<void> {
```

Comment:
This wasn't deprecated?

Reply:
`Event.cancelAndResendKeyRequest` is only used by

https://github.com/matrix-org/matrix-js-sdk/blob/7d687533e753f02166a3014dcbddce07a8d1a4eb/src/client.ts#L3211-L3225

Which was deprecated and deleted in this PR. `Crypto`is also deprecated, this function can't live with `Crypto`.

Reply:
Right, but it should be deprecated before removal, just because another method with a similar name was deprecated does not mean this was deprecated.